### DISCUSSION TITLE
Customize user settings programmatically

### DISCUSSION THREAD

**Discussion by dduong1603** [dagster-io/dagster#17601] [Q&A]
Is there a way to customize the user settings with deployment code? I want to enable some of these as default instead of having the users toggle them on their own
<img width="482" alt="Screenshot 2023-11-01 at 3 08 00 PM" src="https://github.com/dagster-io/dagster/assets/29114884/9f2f4a6a-40a8-49ec-a140-c28da2919940">

**Comment by sryza** [comment]
Hey @dduong1603 - I'm going to convert this to an issue to track the request

**Comment by sryza** [comment]
Issue link: https://github.com/dagster-io/dagster/issues/17614
